marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/entities/search/impl/CollectionFacetHandlerTest.java

```diff
@@ -0,0 +1,31 @@
+package com.marklogic.hub.central.entities.search.impl;
+
+import com.marklogic.client.query.StructuredQueryBuilder;
+import com.marklogic.client.query.StructuredQueryDefinition;
+import com.marklogic.hub.central.entities.search.models.DocSearchQueryInfo;
+import com.marklogic.rest.util.Fragment;
+import org.jdom2.Namespace;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CollectionFacetHandlerTest {
+
+    @Test
+    public void testCollectionFacet() {
+        String uri = "myCollection";
+        String constraintName = "Collection";
+        DocSearchQueryInfo.FacetData facetData = new DocSearchQueryInfo.FacetData();
+        facetData.setStringValues(Collections.singletonList(uri));
+        CollectionFacetHandler collectionFacetHandler = new CollectionFacetHandler();
+        StructuredQueryBuilder queryBuilder = new StructuredQueryBuilder();
+
+        StructuredQueryDefinition queryDefinition = collectionFacetHandler.buildQuery(facetData, queryBuilder);
+        Fragment fragment = new Fragment(queryDefinition.serialize(), Namespace.getNamespace("sch", "http://marklogic.com/appservices/search"));
```

Comment:
Sorry for the nit-pick here, but can you use "search" instead of "sch"? "search" is widely used in ML code as the prefix for this namespace.

Reply:
Sure.